core/templates/pages/blog-home-page/blog-home-page.component.html

```diff
@@ -86,7 +86,7 @@
         <picture>
           <source type="image/webp" [srcset]="getStaticImageUrl('/images/general/no_explorations_found.webp')">
           <source type="image/png" [srcset]="getStaticImageUrl('/images/general/no_explorations_found.png')">
-          <img [src]="getStaticImageUrl('/general/no_explorations_found.png')">
+          <img  [src]="getStaticImageUrl('/general/no_explorations_found.png')">
         </picture>
         <p class="text-center">
           {{ 'I18N_BLOG_HOME_PAGE_NO_RESULTS_FOUND' | translate }}
```

Comment:
In the UI, please drop the per-word capitalization on this, it is not necessary.

Reply:
i have already removed the per-word capitalization property in the line no. 43 of core/templates/pages/blog-home-page/blog-home-page.component.css

Reply:
i have already removed the per-word capitalization property in the line no. 43 of core/templates/pages/blog-home-page/blog-home-page.component.css

Reply:
Done